lib/index.js

```diff
@@ -3,7 +3,6 @@
 module.exports = {
 	configs: {
 		"getify-says": {
-			plugins: [ "@getify/proper-arrows", ],
```

Comment:
it was breaking my eslint server in webstorm, because in flat confing plugins is not an array anymore but an object
I had to define plugins now with
```typescript
import arrows from "@getify/eslint-plugin-proper-arrows";
{
      plugins: {
          "@getify/proper-arrows": arrows,
      }
}
```